wallet-packaging/src/main/assemblies/assembly.xml

```diff
@@ -40,6 +40,9 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
       <includes>
         <include>*:*:jar</include>
       </includes>
+      <excludes>
+        <exclude>org.bouncycastle:bcprov-jdk15on:jar</exclude>
+      </excludes>
```

Comment:
```suggestion
```
Add this exclusion in POM instead of here to exclude its transitive dependency as well, else only the jar will be excluded and not its transitive dependencies.

Reply:
Done